docs/errors/E0423.md

```diff
@@ -0,0 +1,29 @@
+# E0423: missing fallthrough comment in switch case
```

Comment:
"missing fallthrough comment" is good, but if the programmer intended to write a 'break', this diagnostic message is misleading.

Maybe this is a better message:
> missing 'break;' or '// fallthrough' comment between statement and 'case'

For reference, here is the message from [ESLint's no-fallthrough rule](https://eslint.org/docs/latest/rules/no-fallthrough):
> Expected a 'break' statement before 'case'.

Reply:
Sounds good. 
Switched to `missing 'break;' or '// fallthrough' comment between statement and 'case'`
FYI: Also had to switch diag code to E0427 since E0423 was used recently.